requirements.txt

```diff
@@ -1,4 +1,5 @@
 # --------- pytorch --------- #
+python>=3.10,<3.11
```

Comment:
Package setup fails on this line

Reply:
Can you give more information? Or rather, can you tell me how you tested for this?

Reply:
Sorry for confusion. Check out github actions tests
<img width="1080" alt="image" src="https://github.com/ashleve/lightning-hydra-template/assets/43458894/09cf15b0-e22a-416a-b897-6e0cd48c351b">

Reply:
Apologies for the hold-up, I'll get to this soon! Been a busy several weeks.